host/xtest/aes_gcm_counter_overflow_test_decrypt.h

```diff
@@ -0,0 +1,360 @@
+{ TEE_ALG_AES_GCM, TEE_MODE_DECRYPT, TEE_TYPE_AES,
```

Comment:
Could you add a license and copyright notice inline comment with SPDX-License-Identifer tag?

Reply:
done